package.json

```diff
@@ -90,5 +90,8 @@
   ],
   "publishConfig": {
     "@subugoe:registry": "https://gitlab.gwdg.de/api/v4/projects/10921/packages/npm/"
+  },
+  "dependencies": {
```

Comment:
Could you move that to devDependencies pls?

Reply:
yes, did it